docs/tutorial/quickstart.md

```diff
@@ -157,7 +157,8 @@ We can now access our API, both from the command-line, using tools like `curl`..
 
 Or using the [httpie][httpie], command line tool...
 
-    bash: http -a username:password123 http://127.0.0.1:8000/users/
+    bash: pip install httpie      # if you have not installed it yet
+    bash: http -a admin:password123 http://127.0.0.1:8000/users/
```

Comment:
The link for httpie provides more informations to install it. I'd rather leave people choose their installation method.